### PR TITLE
Adding AzureReachabilityReport and AvailableProvidersList APIs to Net…

### PR DESCRIPTION
**Pull request by irrogozh**
…workWatcher swagger + examples

This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/network/resource-manager/Microsoft.Network/2017-08-01/networkWatcher.json [hunk 1]
```diff
@@ -2016,7 +2132,234 @@
...
+    "AzureReachabilityReportLatencyInfo": {
+      "properties": {
+        "timeStamp": {
+          "type": "string",
+          "format": "date-time",
+          "description": "The time stamp."
+        },
+        "score": {
+          "type": "integer",
+          "description": "The relative latency score between 1 and 100, higher values indicating a faster connection."
```
Comment (amarzavery) [inline specification/network/resource-manager/Microsoft.Network/2017-08-01/networkWatcher.json]:
```json5
"minimum": 1,
"maximum": 100
```

Comment (irrogozh) [reply specification/network/resource-manager/Microsoft.Network/2017-08-01/networkWatcher.json]: Done

**Comment by sergey-shandar** [comment]
@irrogozh I'm reviewing the PR and I'll publish my review soon.

**Comment by sergey-shandar** [comment]
@ravbhatnagar new api version

**Comment by ravbhatnagar** [comment]
Looks fine. ARM sign off!

**Comment by sergey-shandar** [comment]
@irrogozh I'm trying to clone your branch but GitHub shows that your branch is from `unknown repository`. Please, contact me over Skype when you can.

**Comment by sergey-shandar** [comment]
As discussed offline, @irrogozh will send a new PR and manually assign me. So I'm closing this PR.

**Comment by irrogozh** [comment]
@sergey-shandar , please, close this pr.

**Comment by sergey-shandar** [comment]
https://github.com/isaacs/github/issues/168

**Pull request closed without merging**